.circleci/config.yml

```diff
@@ -208,7 +208,7 @@ commands:
       - run:
           name: Start internal CI release flow
           command: |
-            if [[ $CIRCLE_TAG == << pipeline.parameters.git_release_tag >> ]]; then
+            if [[ $CIRCLE_TAG =~ << pipeline.parameters.git_release_tag >> ]]; then
```

Comment:
Hmm, but how did it work previously? We've had regex for quite a long time it seems

Reply:
Ah, or it had worked with simple wildcard but when we've switched to the full regexps it stopped right

Reply:
Need to run some real-word test though.. probably try it out in the next beta release.